site/posts/intake-esm-2020-3-16.ipynb

```diff
@@ -0,0 +1,781 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I'm pleased to announce the release of intake-esm version 2020.3.16. This is a new release with bug fixes and new features. Everyone is invited to give it a try, and make their thoughts, suggestions, feedback known! This blogpost outlines these changes. Full changelog is available [here](https://intake-esm.readthedocs.io/en/latest/changelog.html#intake-esm-v2020-03-16).\n",
+    "\n",
+    "\n",
+    "On GitHub: https://github.com/NCAR/intake-esm\n",
+    "\n",
+    "Documentation: https://intake-esm.readthedocs.io/\n",
+    "\n",
+    "<!-- TEASER_END -->\n",
+    "\n",
+    "## Installation\n",
+    "\n",
+    "Intake-esm can be installed from PyPI with pip:\n",
+    "\n",
+    "```bash\n",
+    "python -m pip intall intake-esm --upgrade\n",
+    "```\n",
+    "\n",
+    "It is also available from conda-forge channel for conda isntallations:\n",
+    "\n",
+    "```bash\n",
+    "conda install -c conda-forge intake-esm\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## New Features\n",
+    "\n",
+    "\n",
+    "### Enhanced search: enforce query criteria via `require_all_on` argument\n",
+    "\n",
+    "By default intake-esm's search() method returns entries that fullfill any of the criteria specified in the query. For example, say you define a set of experiments (`experiment_id=['piControl', 'historical']`), and you query the catalog. intake-esm may return a bunch of assets, e.g. `[ModelA.piControl, ModelB.historical, ModelC.piControl, ModelC.historical]`. For some use cases, this may not be a problem at all, however, some analyses may require to exclusively have both experiments. \n",
+    "\n",
+    "Today intake-esm can return entries that fullfill all query criteria when the user supplies the `require_all_on` argument. `require_all_on` consits of attributes/dataframe columns to use when enforcing the query criteria.\n",
+    "\n",
+    "For example: `col.search(experiment_id=['piControl', 'historical'], require_all_on='source_id')` would return all assets with `source_id` that has both `piControl` and `historical` experiments."
```

Comment:
Maybe I would expand this example a bit and use it as the example above.  Then you might not need a "real-world" example.